rust/connlib/clients/shared/src/messages.rs

```diff
@@ -10,6 +10,8 @@ pub struct InitClient {
     pub interface: Interface,
     #[serde(default)]
     pub resources: Vec<ResourceDescription>,
+    #[serde(default)]
+    pub relays: Vec<Relay>,
```

Comment:
cc @AndrewDryga I am assuming the field in the `init` message will be called `relays`.